### ISSUE TITLE
How is this different than the Music App?

### ISSUE THREAD

**Issue by jcklpe** [Rello/audioplayer#353]
I'm having trouble telling how these two apps differ substantially?

**Comment by Rello** [comment]
Hello,
different in the details - my personal opinion:

- Music is simpler and easier in the frontend (less options) and has an Ampache backend
- AP has more flexibility (categories & playlists / if desired with sidebar shown) and has a SONOS backend
- AP supports more formats + webstreams (m3u)
- AP Android native app is in dev by a volunteer

Give both a try and decide for yourself.
If you decide for AP I am happy to receive feedback for further suggestions

**Comment by jcklpe** [comment]
Thanks for the information @Rello  that was very helpful :)

**Comment by Rello** [comment]
at the end I am still interested in your decision and the reason - whatever way you go :-)

**Comment by jcklpe** [comment]
I'll be sure to let you know. I haven't yet started using NextCloud directly for my media library yet but the option to have a dedicated music app on my phone would be a big thing, espc if it had offline play features similar to Spotify.

**Comment by GuyPaddock** [comment]
I just took a look at both https://github.com/owncloud/music and this plugin, and chose the OwnCloud Music app for really only one reason: I like that it has the audio player interface shown at the bottom of the page when you are playing an audio file:
![image](https://user-images.githubusercontent.com/2631799/51656710-322b2f80-1f70-11e9-8922-56d48bb258a8.png)

I also feel like the OwnCloud Music App has a cleaner-looking player (specifically the icons and volume control look a little more clean).

Besides these cosmetic things, I liked everything else about https://github.com/Rello/audioplayer.

I was about to open an issue requesting some documentation on the history and comparisons between the projects, and then I found this related issue.
